share/jupyterhub/templates/admin.html

```diff
@@ -18,11 +18,12 @@
 {% block main %}
 
 <div class="container">
+  {{ pagination.info|safe }}
   <table class="table table-striped">
     <thead>
       <tr>
         {% block thead %}
-        {{ th("User (%i)" % users|length, 'name') }}
+        {{ th("User", 'name') }}
```

Comment:
This can preserve the user count by adding it as its own field, rather than using `users | length`. We don't want to lose the info

Reply:
In this case, I removed it because it's shown similarly in `{{ pagination.info|safe }}`. I can check it to be sure.